example/package.json

```diff
@@ -19,10 +19,11 @@
   },
   "scripts": {
     "start": "lc39 ./index.js",
-    "start:local": "npm start -- --env-path ./local.env"
+    "start:local": "npm start -- --env-path ./local.env --enable-tracing true"
   },
   "dependencies": {
-    "@mia-platform/lc39": "mia-platform/lc39",
+    "@mia-platform/lc39": "../",
```

Comment:
why this change?

Reply:
To point directly to the local version of lc39